lib/fluent/plugin/in_tail.rb

```diff
@@ -356,13 +362,17 @@ def wrap_receive_lines(lines)
       end
 
       def attach(loop)
-        @timer_trigger.attach(loop)
+        if ! @disable_watch_timer
+          @timer_trigger.attach(loop)
+        end
         @stat_trigger.attach(loop)
         on_notify
       end
 
       def detach
-        @timer_trigger.detach if @timer_trigger.attached?
+        if ! @disable_watch_timer
```

Comment:
Sure @repeatedly ... I'll push a new version in a few minutes with the requested changes.